### DISCUSSION TITLE
Is there a way to enable this extension for pages from local net or file?

### DISCUSSION THREAD

**Discussion by sneetsher** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#65] [Q&A] [answered]
I am new user.  The extension is active and shows up in pop menu only in external sites. 

Is there any way to enable it on local net sites and files? I mean for `http://hostname/*`  (no domain extension) and for `file://path/`

**Environment:**
OS: Windows 10
Browser Edge 108.0.1462.54 (Official build) (64-bit)
Extension Version 7.2

**Comment by YoshifumiFuyuno** [comment]
Hi.

* For localhost
I cannot reproduce the problem with `http://localhost`.

* For local files
To use extensions with local files in Chromium-based browsers
`chrome://extensions/` -> "Search from Popup or ContextMenu" -> "Allow access to file URLs" must be enabled.

Best regards

![Animation](https://user-images.githubusercontent.com/9511854/210518284-7111978e-7ba4-4932-a982-a3b215301213.gif)

**Comment by sneetsher** [comment]
Thank you @YoshifumiFuyuno for the quick reply.

I set that option for local files, i tested more, I was wrong about my previous assumption. It seems nothing related to urls but may be page technology/code.

- I get this menu when extension is working.

  [Normal context menu](https://imgur.com/j4ML5Av)

- Other pages where it is not working, i get this kind of legacy menu.

  [Alternative menu](https://imgur.com/zcADCUJ)

  All sites with this kind of menu are private/local. I will look around and test, may be I can find a public site, so you can check it too.

**Comment by YoshifumiFuyuno** [comment, marked as answer]
Perhaps it is being displayed in IE mode?
If you deactivate IE mode, you should be able to see the Edge context menu. If you are using a company PC, you may need permission from the administrator.
![IEMode](https://user-images.githubusercontent.com/9511854/210745660-270b0bcf-8a33-49ac-a5a4-58298ce8fc10.png)

**Reply by sneetsher** [reply]
Yes, it is the IE mode. I couldn't diable it, may be enforced.

Thank you @YoshifumiFuyuno .
